gems/smithy/lib/smithy/welds/endpoints.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Welds
+    # Provides default endpoint builtin/function bindings.
+    class Endpoints < Weld
+      def preprocess(model)
+        model['shapes'].select { |_k, s| s['type'] == 'service' }.each_value do |shape|
+          add_default_endpoints(shape['traits']) unless shape['traits']['smithy.rules#endpointRuleSet']
+        end
+      end
+
+      def endpoint_built_in_bindings
+        {
+          'SDK::Endpoint' => {
+            # Text indenting is used in generated view.
```

Comment:
It would be better if indenting was relative instead of absolute here. So, using `<<~ADD_OPTION` with one level of indenting, then in our view, apply the current indenting level plus this one level.

Reply:
I'm struggling with this because of the multiline text.  Is there a way to have erb views indent multiline text?